content/4_Gold/Tree_Euler.mdx

```diff
@@ -715,6 +715,71 @@ preprocessing and query in $\mathcal{O}(1)$ time.
 
 <FocusProblem problem="rmqSample" />
 
+The following is an example implementation of a sparse table and code that
+answers LCA queries. Build time is $\mathcal{O}(N\log N)$, and queries are
+$\mathcal{O}(1)$.
```

Comment:
honestly i'm not sure if this is really needed

Reply:
hmm, true. i just wrote one for fun and decided to PR it, so i can close if we don't need it

Reply:
wait what am i saying this literally the solution to a focus problem my bad g